pages/concepts/ai-engine/ai-engine-personalities.mdx

```diff
@@ -0,0 +1,58 @@
+# AI Engine Personalities
+
+## Introduction
+
+This guide explores the **AI Engine personalities** within Fetch.ai's ecosystem. Let's delve into how these personalities enhance user interactions when operating with Ai Engine and DeltaV!
+
+## The AI Engine personalities: Powering Fetch.ai's Intelligence
+
+The [AI Engine ↗️](/concepts/ai-engine/ai-engine-intro) stands as the heart of Fetch.ai's AI infrastructure, translating human language inputs into actionable tasks for a network of [Agents ↗️](/guides/agents/getting-started/whats-an-agent) to fulfill them. The AI Engine bridges the gap between users and AI by leveraging powerful Large Language Models (LLMs) to understand complex requests. Here's how it works:
```

Comment:
We could write "actionable functions" instead of "actionable tasks" and put a hyperlink for the corresponding page in the docs which is about the concept of agent functions.

Reply:
And please replace all the occurrence of tasks with function - just to avoid introducing one more abstraction for the same thing